### ISSUE TITLE
Navbar burger menu doesn't toggle in Firefox

### ISSUE THREAD

**Issue by Tentacular261** [papercss/papercss#173]
# The Issue
The navbar currently hides child links, other than the brand, in a burger menu for mobile devices. Once the burger menu is clicked, it is supposed to open a menu with all of the previously hidden links. However, in Firefox, the burger menu toggle doesn't open the menu:

[Image of burger menu that doesn't open in FF](https://prntscr.com/lycq2e)

It works great for Chrome, but it doesn't work in Firefox. I have tested in both the Responsive Design Mode in the desktop version of FF and on standard FF for my phone. 

If you apply the checked attribute to the hidden `input[type=checkbox]` that papercss uses, then the menu opens like it should:

[Image of opened burger menu in FF after applying checked attribute](https://prntscr.com/lycr5d)

Currently, the way papercss does the navbar is by having a hidden `input[type=checkbox]` followed by a label that has a for attribute naming the checkbox input, except the label is inside of a button:

```html
<input id="collapsible1" type="checkbox" name="collapsible1">
<button>
    <label for="collapsible1">
        <div class="bar1"></div>
        <div class="bar2"></div>
        <div class="bar3"></div>
    </label>
</button>
```

This seems to mean that FF doesn't accept `label[for=checkbox]`s being located inside of `button`s. 

# To reproduce
Go to the navbar documentation site (https://www.getpapercss.com/docs/components/navbar/) in Firefox mobile.

**Comment by Phreshhh** [comment]
Hi!

I maked the problem fix.
In Edge still not working, but soon Edge change its engine to Chromium, so will solve also.
The problem source was the button tag. I cutted off from the html and adjust to it the css.

I tired pull it, but the sh.t happends with the git..  I maked the fix in 2 minutes, but i cant fix the stylelint problem without modify the packages.json file.. but if i modify it, then the travis dont accept it. I waste for this more the 3 hours at now so i give up.

i tired with
"stylelint": "node node_modules/stylelint/bin/stylelint.js 'src/**/*.scss'"
"stylelint": "stylelint/bin/stylelint.js 'src/**/*.scss'"

on my computer works all the 2, but the travis not eat it.
I tired also modify my environment variables is with path mods, but nothing.

So

here is the files, build in please, if you want this fix.
docs/content/docs/components/navbar.md
src/components/_navbar.scss

[papercss-ff-fix_included_builded_files.zip](https://github.com/papercss/papercss/files/2807772/papercss-ff-fix_included_builded_files.zip)

Have a nice day guys and thanks for the PaperCSS! :)

**Comment by TotomInc** [comment]
This issue is actually being fixed in #179.

**Comment by rhyneav** [comment]
Fixed and deployed in version 1.7.0!
